e2e-tests/gef/cypress/integration/ineligible-gef.spec.js

```diff
@@ -23,9 +24,9 @@ context('Ineligible GEF Page', () => {
   });
 
   describe('Clicking on Back Button', () => {
-    it('redirects user to ** page', () => {
+    it('redirects user to the applications and notices page', () => {
       ineligibleGef.backLink().click();
-      cy.url().should('eq', relative('/gef/ineligible-gef'));
+      cy.url().should('eq', relative('/dashboard/0'));
```

Comment:
should this link to the GEF dashboard instead? `/dashboard/gef/0`

Reply:
Cheers .. dint know this was possible. Will update now :+1: